teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < person
+  def initialize(specialization, age, name, parents_permission)

```

Comment:
- [ ] Kindly make sure you test your app before submitting the PR. as you may notice you did not spell correctly the parent class `Person` so There would be no way the app can work.
`Hint:`
```rb
class Teacher < Person
```

![image](https://user-images.githubusercontent.com/97100091/213209720-e3372e78-12c9-4490-9de0-05c1e4f45d30.png)